Headline

jazz-784 opened the pull request:
my second project Mobile-Portfolio 
added feature branch 
worked on linters config
made html and css files from sratch

jazz-784 committed:
html page added

jazz-784 committed:
css page added

jazz-784 committed:
updated

franclobo reviewed (commented):
Hi @jazz-784, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that hasn't used the correct README template, kindly use this [README template](https://github.com/microverseinc/readme-template) and remember that all main headers should be present in your repo.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

jazz-784 committed:
Update README.md

jazz-784 committed:
updated read me file

updated as reviwed invalid from reviewer

DuaneDave reviewed (commented):
Hi @jazz-784 👋 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is in a new branch. Kindly submit the old branch

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

abdulhamiid commented on index.html:
```diff
@@ -0,0 +1,67 @@
+                <p>
+                    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don't hestiate to contact me.
+                </p>
+// just added the comments to save and modfy 
+                <ul>
```
Please let's remove this text as it is not part of the Figma design and it is important that we stick to the design as much as possible

![image](https://user-images.githubusercontent.com/78251927/206823639-0e59f8a3-51cc-4d18-a2c0-9243acfa9661.png)

abdulhamiid commented on index.html:
```diff
@@ -0,0 +1,67 @@
+            <a href="#">Welcome</a>         
+        </nav>
+        <nav class="union-one">
+            <a>
+                <img src="./images/Icon - Menu.png" alt="">
+            </a>
```
[OPTIONAL] It will be great if we use a [Hamburger icon](https://fontawesome.com/icons/bars?s=solid&f=classic) here for semantics. ✔

```
<a>
  <i class="fa-solid fa-bars"></i>
</a>
```

abdulhamiid reviewed (changes requested):
# Status: Required Changes :recycle:
Hi @jazz-784 👋 
![](https://media.giphy.com/media/UdaSUYClr92cjWpeVx/giphy.gif)
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Required Changes ♻️
_Please kindly check the comments under the review._

### Your commit messages :point_down: 

![image](https://user-images.githubusercontent.com/78251927/206824197-25561f60-b582-4ae0-9a6b-fd968f507402.png)

 As part of [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md), the commit message should be written in the imperative tense and capitalized, 

- [ ] Example :point_down: 

:x: Bad commit
```
html page added
```
:heavy_check_mark: Good commit
```
Add html page
```

- [ ] You wrote all the commit messages in lowercase. ❌ 

- [ ] Commit message should be Capitalized. ✔️ 
- [ ] Commit message should be written in the imperative tense ✔️ and not past tense ❌ 

- [ ]  You can learn more about **Professional repo - basic rules**  :point_right: [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

- [ ] **IMPORTANT:** as it is not easy to change commit history, please kindly correct this behavior in the future.

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

jazz-784 committed:
updated and modified index.html

all changes updated as suggested by the reviewer and also would work on my commit history

Kamba56 commented on index.html:
```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="  https://developers.google.com/fonts/docs/getting_started">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav class="logo">
+            <a href="#">Welcome</a>         
+        </nav>
+        <nav class="union-one">
+            <a>
+                <i class="fa-solid fa-bars"></i>
+            </a>
+        </nav>
+    </header>
+    <section id="headline-section">
+        <div class="headline-container">
+            <h1>Hey there. I'm Mariane <br>
+                <span>I'm a software developer </span></h1>
+                <p>
```
[Optional] Please use an indentation of two spaces in all your documents. This is a convention and also [best practice.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

Kamba56 reviewed (approved):
Hi @jazz-784 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
## Highlights
- [x] All linters passed!👏👏👏
- [x] Great use of Github workflow!👏👏👏
- [x] Descriptive PR!👏👏👏

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
